CHANGELOG.md

```diff
@@ -16,12 +15,17 @@
 ### `Fixed`
 
 - Typo in previous release date
+- [#208](https://github.com/nf-core/mhcquant/pull/208) - nf-core template update (version 2.3.2)
```

Comment:
I doubt that. It is not appearing in the list of changes and https://github.com/nf-core/mhcquant/pull/204 should be closed after you have synced the template update into dev. Did you follow https://nf-co.re/developers/sync#merging-automated-prs?